src/components/MetadataSidebar.vue

```diff
@@ -26,6 +26,7 @@
     </div>
     <div v-if="isMetadataExtracted" id="dicom-metadata-sidebar-content" class="oc-p-s">
       <table class="details-table">
+        <caption>Details table</caption>
```

Comment:
will this be displayed in the UI or is this just for improving accessibility? If it's for screen reader, maybe write something that is more descriptive, like "DICOM metadata details table" or similar

Reply:
I updated the text as description in a <p> element, but possibly is visible from UI. If this isn't intended let me know to remove it completely until a new solution comes up

Reply:
can you explain me what's your reasoning for adding the captions title?